### DISCUSSION TITLE
ValueError: need at least one array to concatenate

### DISCUSSION THREAD

**Discussion by SuperFDX** [fcakyon/small-object-detection-benchmark#22] [Q&A]
Hi @ fcakyon, I encountered some problems about training details during the training. I'm trying to reproduce the experiment. I want to use vfnet, so I use the code in the following path：
https://github.com/fcakyon/small-object-detection-benchmark/blob/main/mmdet_configs/visdrone_vfnet/vfnet_crop_480_960_cls_60.py
and I use visdrone_ to_ Coco.py converts the visdrone training and verification split into the coco format and uses https://github.com/fcakyon/sahi-benchmark/blob/mmdet_ Tools/train.py conducted training, but reported an error ValueError: need at least one array to concatenate. I went online to find a solution, and the answer was that the first possible path was wrong, which has been ruled out. Second, there is a problem with the converted json file. It is said that the name field in "categories" may not be the category name, but I have seen that the converted json file has no problem. How can I solve it? I look forward to your reply.

**Comment by fcakyon** [comment]
Have you checked the package versions? I am not able to recreate this error.

**Reply by SuperFDX** [reply]
Thank you for your reply. The main package versions I use are as follows:
sahi                    0.11.4
setuptools              58.0.4
Shapely                 1.8.5.post1
torch                   1.10.0
detectron2              0.6+cu113
mmcv-full               1.4.3
mmdet                   2.21.0
opencv-python           4.6.0.66
numpy                   1.23.5

**Reply by fcakyon** [reply]
Maybe you can try it with newer mmdet and mmcv versions and report back if its fixed

**Reply by SuperFDX** [reply]
OK, thank you for your reply. I'll try it.

**Reply by SuperFDX** [reply]
Hi @fcakyon , now that I have used mmdet==2.26.0, mmcv full==1.7.0 and torch==1.10.0, the same error still occurs. Do you have any solutions? We look forward to your reply.
